### PR TITLE
feat(nodes): Add toggle to WakuNodesModal

### PR DESCRIPTION
**Pull request by borismelnik**
Closes: #5000

### What does the PR do

Add use mail servers setting

### Affected areas

WakuNodesModal

### CODE REVIEW

File: ui/app/AppLayouts/Profile/popups/WakuNodesModal.qml [hunk 1]
```diff
@@ -46,19 +46,19 @@ StatusModal {
             anchors.rightMargin: Style.current.padding
 
             // TODO re-add that when the status-go API to disconnect from the mailserver is added
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/popups/WakuNodesModal.qml:48]: You can get rid of the TODO now

File: ui/app/AppLayouts/Profile/popups/WakuNodesModal.qml [hunk 2]
```diff
@@ -46,19 +46,19 @@ StatusModal {
...
+                anchors.leftMargin: -Style.current.padding
+                anchors.right: parent.right
+                anchors.rightMargin: -Style.current.padding
+                title: qsTr("Use Waku nodes")
+                components: [
+                    StatusSwitch {
+                        checked: root.messagingStore.useMailservers
+                        onCheckedChanged: root.messagingStore.toggleUseMailservers(checked)
+                    }
+                ]
```
Comment (jrainville) [inline ui/app/AppLayouts/Profile/popups/WakuNodesModal.qml:60]: You might need the `sensor.onClicked` to be able to click the ListItem as well

**Review by jrainville — COMMENTED** [review]
Nice work. Some small comments

**Review by richard-ramos — APPROVED** [review]

**Review by jrainville — APPROVED** [review]

**Review by iurimatias — APPROVED** [review]
Screen Shot 2022-03-29 at 4 58 40 PM

**Pull request merged**